velox/vector/fuzzer/VectorFuzzer.cpp

```diff
@@ -243,6 +243,12 @@ VectorPtr
 VectorFuzzer::fuzz(const TypePtr& type, vector_size_t size, bool flatEncoding) {
   VectorPtr vector;
 
+  vector_size_t outputSize = size;
```

Comment:
nit: I'd use a new local var actualSize= size and change actualSize instead of function input size.